README.md

```diff
@@ -1,4 +1,147 @@
 [![CI](https://github.com/aoscloud/aos_core_iam_cpp/workflows/Build/badge.svg)](https://github.com/aoscloud/aos_core_iam_cpp/actions?query=workflow%3ABuild)
 [![codecov](https://codecov.io/github/aoscloud/aos_core_iam_cpp/branch/main/graph/badge.svg?token=kg8h7ATd9S)](https://codecov.io/github/aoscloud/aos_core_iam_cpp)
 
-# aos_core_iam_cpp
+# Aos core IAM manager
+
+## Prepare build environment
+
+```sh
+sudo apt install lcov libsofthsm2 libsystemd-dev
+pip install conan
+```
+
+# Build for host
```

Comment:
Should be only one main header. Please install `markdownlint` for VSCode and check.